includes/admin/post-types/meta-boxes/class.llms.meta.box.course.options.php

```diff
@@ -328,7 +328,51 @@ public function get_fields() {
 						'label'            => __( 'Choose Prerequisite Course Track', 'lifterlms' ),
 						'value'            => $course_tracks,
 					),
-
+					array(
+						'type'          => 'checkbox',
+						'label'         => __( 'Enable Lesson Drip', 'lifterlms' ),
+						'desc'          => __( 'Set global drip restrictions so lesson content becomes available at an interval you define for the course.', 'lifterlms' ),
+						'id'            => $this->prefix . 'lesson_drip',
+						'is_controller' => true,
+						'value'         => 'yes',
+						'class'         => '',
+						'desc_class'    => 'd-3of4 t-3of4 m-1of2',
+					),
+					array(
+						'class'            => 'llms-select2',
+						'controller'       => '#' . $this->prefix . 'lesson_drip',
+						'controller_value' => 'yes',
+						'is_controller'    => true,
+						'type'             => 'select',
+						'id'               => $this->prefix . 'drip_method',
+						'label'            => __( 'Drip Method', 'lifterlms' ),
+						'value'            => array(
+							array(
+								'key' => 'start',
+								'title' => __( 'After course start', 'lifterlms' ),
```

Comment:
```suggestion
								'title' => __( 'After course start or enrollment', 'lifterlms' ),
```
Technically if there is no start date for the course, this method falls back to using the student's enrollment date. So let's update this label to make that a bit more clear.

Saying "this or that" in the label now begs the question, when is it start and when is it enrollment. A hint (desc) could help.